tealish/expression_nodes.py

```diff
@@ -130,6 +137,14 @@ def __init__(
     def process(self) -> None:
         self.a.process()
         self.b.process()
+
+        print(self.a.tealish_type())
+        print(self.b.tealish_type())
+        print(self.op)
+        if self.a.tealish_type() == TealishType.bigint:
+            self.t_type = TealishType.bigint
+            self.op = "b" + self.op
```

Comment:
obv would need something way less terrible here to map to the right ops

Reply:
possibly something between the raw langspec parsing and the op lookup that considers `TealishType` && `op` and returns a more specific op given the TealishType